main.yml

```diff
@@ -3,9 +3,14 @@
   remote_user: root
   gather_facts: yes
 
+- hosts: test
```

Comment:
I will delete this line as there already exists hosts key word at the beginning of the file.